backend/src/Designer/Controllers/DatamodelsController.cs

```diff
@@ -217,5 +226,42 @@ private static string GetFileNameFromUploadedFile(IFormFile thefile)
         {
             return ContentDispositionHeaderValue.Parse(new StringSegment(thefile.ContentDisposition)).FileName.ToString();
         }
+
+        private bool TryValidateSchema(string schema, out ValidationProblemDetails problemDetails)
```

Comment:
I'm not that familiar with the `out` usage so I only ask this based on curiousity;
What will the value of problemDetails be if the schema has no issues and the function returns `true`? The way I read the code it looks like it will return the problemDetails created at line 243, but I guess there is some magic happing here that sets it to something else 😅

Reply:
You're right. It should be null if it's valid. Will fix it.